### PR TITLE
LRDOCS-6299 Migrating an Angular app to Liferay

### PR DESCRIPTION
**Pull request by jhinkey**
This branch provides the following appdev files:

- how-to article _Migrating an Angular App to Liferay Portal_ https://issues.liferay.com/browse/LRDOCS-6299
- placeholder intros and folders for ALL the UI types (React, Vue, JSF, … etc.)
- code for the Angular guestbook app before and after migration to Liferay. https://issues.liferay.com/browse/LRDOCS-6229

@mwilliams2014 wrote the original article and created the app and migrated it.

### CODE REVIEW

File: developer/appdev/articles/01-ui-development/02-angular-widget/02-migrating-an-angular-app-to-product.markdown [hunk 1]
```diff
@@ -202,14 +205,14 @@ Follow these steps:
...
-11. Finally, deploy your portlet bundle:
+11. Finally, deploy your bundle:
 
         npm run deploy
 
 Congratulations! Your Angular app is deployed and now available as a widget that
 you can add to site pages. 
 
-The liferay-npm-bundler confirms the deployment:
+The Liferay NPM Bundler confirms the deployment:
```
Comment (mwilliams2014) [inline developer/appdev/articles/01-ui-development/02-angular-widget/02-migrating-an-angular-app-to-product.markdown]: @jhinkey we've always referred to it as `liferay-npm-bundler` in the docs. That is the npm package name. If you think Liferay npm Bundler sounds better, that's fine, but we should update it across all the docs. If we do go this route, never capitalize npm, as they don't capitalize it themselves.

Comment (jhinkey) [reply developer/appdev/articles/01-ui-development/02-angular-widget/02-migrating-an-angular-app-to-product.markdown]: @mwilliams2014 It's awkward to refer to Liferay Bundler Generator in regular text and then the bundler by package name liferay-npm-bundler. I agree to use lowercase npm. I'll update that.

Comment (mwilliams2014) [reply developer/appdev/articles/01-ui-development/02-angular-widget/02-migrating-an-angular-app-to-product.markdown]: Sounds good. I'll make the change across the rest of our docs

**Comment by jhinkey** [comment]
I'll send this and the UI dev intro in a new PR.

**Pull request closed without merging**